x-pack/plugins/event_log/server/es/cluster_client_adapter.ts

```diff
@@ -313,8 +297,153 @@ export class ClusterClientAdapter {
+            [
+              {
+                nested: {
+                  path: 'kibana.saved_objects',
+                  query: {
+                    bool: {
+                      must: [
+                        {
+                          term: {
+                            'kibana.saved_objects.rel': {
+                              value: SAVED_OBJECT_REL_PRIMARY,
+                            },
+                          },
+                        },
+                        {
+                          term: {
+                            'kibana.saved_objects.type': {
+                              value: type,
+                            },
+                          },
+                        },
+                        {
+                          terms: {
+                            // default maximum of 65,536 terms, configurable by index.max_terms_count
+                            'kibana.saved_objects.id': ids,
+                          },
+                        },
+                        namespaceQuery,
+                      ],
+                    },
+                  },
+                },
+              },
+              start && {
+                range: {
+                  '@timestamp': {
+                    gte: start,
+                  },
+                },
+              },
+              end && {
+                range: {
+                  '@timestamp': {
+                    lte: end,
+                  },
+                },
+              },
+            ],
+            isUndefined
+          ),
+        },
+      },
+      aggs: {
+        saved_objects: {
+          nested: {
+            path: 'kibana.saved_objects',
+          },
+          aggs: {
+            saved_object: {
+              filter: {
+                terms: {
+                  'kibana.saved_objects.id': ids,
+                },
+              },
+              aggs: {
+                ids: {
+                  // use 'terms' with size of MAX_BUCKETS_LIMIT instead of 'composite', because of filter
+                  terms: {
+                    field: 'kibana.saved_objects.id',
+                    size: MAX_BUCKETS_LIMIT,
+                  },
+                  aggs: {
+                    summary: {
+                      reverse_nested: {},
+                      aggs,
+                    },
+                  },
+                },
+              },
+            },
+          },
+        },
+      },
+    };
+
+    try {
+      const result = await esClient.search({
+        index,
+        body,
+      });
+      return result.body.aggregations.saved_objects.saved_object.ids.buckets.map(
+        (savedObject: Record<string, unknown>) => ({
+          savedObjectId: savedObject.key,
+          summary: savedObject.summary,
+        })
+      );
+    } catch (err) {
+      throw new Error(
+        `querying for Event Log alerts instances summaries by alert ids "${ids}" failed with: ${err.message}`
```

Comment:
This error message refers to "Alert ids", but the api seems to be _any_ Saved Object, not just alerts... am I right?

If so, this message should be changed to be more generic.